src/scss/components/modal/_index.scss

```diff
@@ -31,25 +33,26 @@ hx-modal {
   }
 
   > hx-div {
-    --hxPaddingLeft: @space[md];
-    --hxPaddingRight: @space[md];
+    --hxPaddingLeft: #{map-get($space, md)};
+    --hxPaddingRight: #{map-get($space, md)};
+
     flex-grow: 1;
     order: 2;
 
-    #HXDiv.shadyRoot({
-      padding: 0 @space[md];
-    });
+    @include hxDiv-shadyRoot {
+      padding: 0 map-get($space, md);
+    };
 
     &:first-child {
       margin-top: 4rem;
     }
 
     &:last-child {
-      --hxPaddingBottom: @space[md];
+      --hxPaddingBottom: map-get($space, md);
```

Comment:
Need to explicitly interpolate the value...

```scss
--hxPaddingBottom: #{map-get($space, md)};
```

Reply:
Done the changes.